package.json

```diff
@@ -1,14 +1,18 @@
 {
   "name": "@openstax/highlighter",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "main": "dist/highlighter.js",
   "license": "MIT",
   "scripts": {
-    "typescript-watch": "tsc --noEmit --watch",
-    "webpack-watch": "webpack --config=webpack.config.js --mode=development --watch",
     "build": "webpack --config=webpack.config.js --mode=production",
     "jest-watch": "jest --watchAll",
-    "watch": "npm-run-all jest-watch webpack-watch typescript-watch"
+    "test": "jest --verbose --maxWorkers=2",
+    "type-check": "tsc",
+    "js-lint": "eslint src/**/*.js",
+    "typescript-lint": "tslint -p ./tsconfig.json",
+    "typescript-watch": "tsc --noEmit --watch",
+    "watch": "npm-run-all jest-watch webpack-watch typescript-watch",
+    "webpack-watch": "webpack --config=webpack.config.js --mode=development --watch"
```

Comment:
Similarly, could this be in the `{command}:{subcommand}` notation? 

Aside: I realized just now that the npm commands are `{language}-{command}` with a few exceptions: 

- `type-check`
- `test`
- `build`
- `jest-watch`

Would the `{command}:{subcommand}` notation be more consistent?

- `lint`
  - `lint:ts`
  - `lint:js`
  - `lint:css` (hypothetical)
- `watch`
  - `watch:build`
  - `watch:ts`
  - `watch:test`
  - `watch:css` (hypothetical)
- `build`
- `test`